kolibri/core/assets/src/vue/session-nav-widget/index.vue

```diff
@@ -1,52 +1,85 @@
 <template>
 
-  <nav-bar-item @click="trigger()">
+  <nav-bar-item>
     <div class="wrapper">
-      <div v-if="loggedIn" class='user-icon'>{{ displayText }}</div>
+      <div v-if="loggedIn">
+        <div class='user-icon' id="user-dropdown" @click="showUserDropdown">{{ displayText }}</div>
+      </div>
       <div v-else>
-        <svg role="presentation" height="40" width="40" viewbox="0 0 24 24" src="./person.svg"></svg>
-        <div class="label">Log In</div>
+        <login-modal></login-modal>
       </div>
     </div>
   </nav-bar-item>
 
+  <div id="dropdown" v-show="showDropdown" transition="slide">
```

Comment:
Might be nice to give this a different name - 'drop down' is a common UI widget so this is a bit confusing. Maybe 'pop-up-widget' or something?